test/controllers/account/permissions_controller_test.rb

```diff
@@ -275,14 +275,12 @@ class Account::PermissionsControllerTest < ActionController::TestCase
         },
       )
 
-      current_user.reload
```

Comment:
I'd say "tidy up" is a little subjective! Could we say something like `inline/chain reloading of current user` instead? Possibly with a description as to why we think it's better, which could simply be "fewer lines"